src/HonzaBotner.Discord.Services/Jobs/NewsJobProvider.cs

```diff
@@ -11,9 +11,12 @@
 
 namespace HonzaBotner.Discord.Services.Jobs;
 
-[Cron("0 0 * * * *")]
+// Run every 30 minutes
+[Cron("0 0/30 * * * *")]
 public class NewsJobProvider : IJob
```

Comment:
Is it ok? according to https://crontab.guru/every-30-minutes it should be `*/30 * * * *`